handler/wallet/import.go

```diff
@@ -53,7 +51,7 @@ func importHandler(
 
 	wallet := model.Wallet{
 		ID:         result,
-		Address:    addr.String(),
+		Address:    result[:1] + addr.String()[1:],
```

Comment:
make sure neither of these are empty so that we don't potentially panic here?

Reply:
the addr.String() is fine because it's already validated. Added address parsing for result